### PR TITLE
feat(teamStats): Add new/returning Issues and Issues Triaged

### PR DESCRIPTION
**Pull request by scttcper**
Adds a shared component that calls an endpoint with different query parameters.

Screen Shot 2021-12-21 at 3 33 04 PM

### CODE REVIEW

File: static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+      prevProps.teamSlug !== teamSlug ||
+      !isEqual(prevProps.projects, projects)
+    ) {
+      this.remountComponent();
+    }
+  }
+
+  renderLoading() {
+    return this.renderBody();
+  }
```
Comment (taylangocmen) [inline static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx:96]: This looks like it's not necessary.

**Review by taylangocmen — APPROVED** [review]
lgtm

File: static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx (hunk 1 above)
Comment (scttcper) [reply static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx:96]: there might be an alwaysRenderBody i forgot about

File: static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        <IssuesChartWrapper>
+          {loading && <Placeholder height="200px" />}
+          {!loading && (
+            <BarChart
+              style={{height: 200}}
+              stacked
+              isGroupedByDate
+              useShortDate
+              legend={{right: 0, top: 0}}
+              xAxis={barAxisLabel(4)}
```
Comment (scttcper) [inline static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx:154]: need to fix this before merging

**Pull request merged**